docs/runtimes/private-repo.ipynb

```diff
@@ -0,0 +1,145 @@
+{
```

Comment:
<p>private repo is more relevant for private git repo and how to authenticate to it, can change to - Using your own git repo as a function hub</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/3190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>